projects/js-packages/publicize-components/src/components/media-picker/index.js

```diff
@@ -27,6 +27,7 @@ const clickHandler = open => e => {
  * @param {object} props.mediaDetails - The details of the media for preview
  * @param {Function} props.onChange - A callback that can be passed to parent for validation
  * @param {string} props.wrapperClassName - A class name to be added to the wrapper
+ * @param {object} props.allowedMediaTypes - Custom allowed media types
```

Comment:
Are we using this? Should this be passed in when auto conversion is not enabled?

Reply:
It's used here: fbhepr%2Skers%2Swrgcnpx%2Scebwrpgf%2Swf%2Qcnpxntrf%2Schoyvpvmr%2Qpbzcbaragf%2Sfep%2Spbzcbaragf%2Sfbpvny%2Qvzntr%2Qtrarengbe%2Scnary%2Szbqny.wf%3Se%3Qro7374p5%26zb%3Q898%26sv%3Q22%2382-og

It was not working before, this fixes it